src/index.js

```diff
@@ -5,6 +5,7 @@ import App from "./App";
 import { makeServer } from "./server";
 import { BrowserRouter } from "react-router-dom";
 import { AuthProvider } from "./src/context/provider/AuthProvider";
+import { VideoProvider } from "./src/context/provider/VideoProvider";
```

Comment:
You can use barrel export here for all the contexts.

Reply:
Ok will try that, Thanks.